### PR TITLE
Add repos as an input to the action

### PR DESCRIPTION
**Pull request by rohitnb**
### What does this branch do?

- The action now has an additional input parameter - `repo`. This input is not mandatory and defaults to the calling repository
- Modifications were required for the action.yml and the invoke-action.yml to demo how it would work

### Intended use-case

- Allow security personnel to fetch reports for any repository by passing repo's as an input. This makes it easier for them too.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ inputs:
     required: true
     description: 'input description here'
     default: 'default value if applicable'
+  repo:
+    required: false
+    description: 'repository name i.e. owner/repo-name format'
+    default: ${{ github.repository }}
```
Comment (rohitnb) [inline action.yml:12]: Still need to test if this default value is accepted. Forgot to test this.

Comment (rohitnb) [reply action.yml:12]: It is.

**Review by amitgupta7 — CHANGES_REQUESTED** [review]
not required.

**Comment by amitgupta7** [comment]
@rohitnb I needed to clean-up some conditional code. So I have taken the liberty to create a separate [check-in ](https://github.com/amitgupta7/ghas-reports-action/commit/c7e27cb22c1dd2e390f42a331908e5acd44df493) Have also created a new [release](https://github.com/amitgupta7/ghas-reports-action/releases/tag/v3.4) for testing this change.

**Pull request closed without merging**